4-storage/dashdb-storage/atomic-def.sql

```diff
@@ -0,0 +1,160 @@
+-- in the CREATE TABLE statement and also in the COMMENT ON TABLE statement.
+
+-- Create the schema
+CREATE SCHEMA atomic;
+
+-- Create events table
+CREATE TABLE atomic.events (
+	app_id VARCHAR(255),
+	platform VARCHAR(255),
+	etl_tstamp TIMESTAMP,
+	collector_tstamp TIMESTAMP not null,
+	dvce_created_tstamp TIMESTAMP,
+	event VARCHAR(128),
+	event_id CHARACTER(36) not null unique,
+	txn_id INTEGER,
+	name_tracker VARCHAR(128),
+	v_tracker VARCHAR(100),
+	v_collector VARCHAR(100) not null,
+	v_etl VARCHAR(100) not null,
+	user_id VARCHAR(255),
+	user_ipaddress VARCHAR(45),
+	user_fingerprint VARCHAR(50),
+	domain_userid VARCHAR(36),
+	domain_sessionidx SMALLINT,
+	network_userid VARCHAR(38),
+	geo_country CHARACTER(2),
+	geo_region CHARACTER(2),
+	geo_city VARCHAR(75),
+	geo_zipcode VARCHAR(15),
+	geo_latitude DOUBLE,
+	geo_longitude DOUBLE,
+	geo_region_name VARCHAR(100),
+	ip_isp VARCHAR(100),
+	ip_organization VARCHAR(100),
+	ip_domain VARCHAR(100),
+	ip_netspeed VARCHAR(100),
+	page_url VARCHAR(4096),
+	page_title VARCHAR(2000),
+	page_referrer VARCHAR(4096),
+	page_urlscheme VARCHAR(16),
+	page_urlhost VARCHAR(255),
+	page_urlport INTEGER,
+	page_urlpath VARCHAR(3000),
+	page_urlquery VARCHAR(6000),
+	page_urlfragment VARCHAR(3000),
+	refr_urlscheme VARCHAR(16),
+	refr_urlhost VARCHAR(255),
+	refr_urlport INTEGER,
+	refr_urlpath VARCHAR(6000),
+	refr_urlquery VARCHAR(6000),
+	refr_urlfragment VARCHAR(3000),
+	refr_medium VARCHAR(25),
+	refr_source VARCHAR(50),
+	refr_term VARCHAR(255),
+	mkt_medium VARCHAR(255),
+	mkt_source VARCHAR(255),
+	mkt_term VARCHAR(255),
+	mkt_content VARCHAR(500),
+	mkt_campaign VARCHAR(255),
+	se_category VARCHAR(1000),
+	se_action VARCHAR(1000),
+	se_label VARCHAR(1000),
+	se_property VARCHAR(1000),
+	se_value DOUBLE,
+	tr_orderid VARCHAR(255),
+	tr_affiliation VARCHAR(255),
+	tr_total DECIMAL(18,2),
+	tr_tax DECIMAL(18,2),
+	tr_shipping DECIMAL(18,2),
+	tr_city VARCHAR(255),
+	tr_state VARCHAR(255),
+	tr_country VARCHAR(255),
+	ti_orderid VARCHAR(255),
+	ti_sku VARCHAR(255),
+	ti_name VARCHAR(255),
+	ti_category VARCHAR(255),
+	ti_price DECIMAL(18,2),
+	ti_quantity INTEGER,
+	pp_xoffset_min INTEGER,
+	pp_xoffset_max INTEGER,
+	pp_yoffset_min INTEGER,
+	pp_yoffset_max INTEGER,
+	useragent VARCHAR(1000),
+	br_name VARCHAR(50),
+	br_family VARCHAR(50),
+	br_version VARCHAR(50),
+	br_type VARCHAR(50),
+	br_renderengine VARCHAR(50),
+	br_lang VARCHAR(255),
+	br_features_pdf SMALLINT,
+	br_features_flash SMALLINT,
+	br_features_java SMALLINT,
+	br_features_director SMALLINT,
+	br_features_quicktime SMALLINT,
+	br_features_realplayer SMALLINT,
+	br_features_windowsmedia SMALLINT,
+	br_features_gears SMALLINT ,
+	br_features_silverlight SMALLINT,
+	br_cookies SMALLINT,
+	br_colordepth VARCHAR(12),
+	br_viewwidth INTEGER,
+	br_viewheight INTEGER,
+	os_name VARCHAR(50),
+	os_family VARCHAR(50),
+	os_manufacturer VARCHAR(50),
+	os_timezone VARCHAR(255),
+	dvce_type VARCHAR(50),
+	dvce_ismobile SMALLINT,
```

Comment:
On disk these are indeed 1 or 0, but the Redshift table definition has them as booleans. Would it be possible to load them into booleans for DashDB?